tripy/tripy/frontend/trace/ops/slice.py

```diff
@@ -256,8 +256,36 @@ def clamp_bound(bound: Union[int, Tensor]) -> Union[int, Tensor]:
     return out
 
 
-# Because the helper is called inside another function, we need to skip one entry in the call stack to find
-# the original call to user code.
-@frontend_utils.convert_to_tensors(skip_num_stack_entries=1)
+@frontend_utils.convert_to_tensors()
 def slice_helper(tensor, *slice_params: TensorLike):
+    from tripy.utils import get_arg_candidate_column_offsets
+
+    # The default behavior of convert_to_tensors will not add the correct column info to the slice params
+    # because this call occurs *inside* the overridden call to __getitem__, so we adjust the column info manually.
+
+    # Look for the stack frame index to __getitem__. We need to go one stack frame beyond to get to the *user* call of __getitem__.
+    # It will be the same for all the slice params
+    frame_index = -1
+    if slice_params:
```

Comment:
What's the case where there are no `slice_params`?

Reply:
I'll change it to an assert. I don't think it's possible for it to be empty but it's best to avoid an indexing error in case it somehow happens.